spyder/widgets/about.py

```diff
@@ -85,11 +120,28 @@ def __init__(self, parent):
             <a href="https://winpython.github.io/">WinPython</a>
             also contribute to this plan.
             </p>
+            </div>""").format(
+                github_url=__project_url__,
+                trouble_url=__trouble_url__,
+                forum_url=__forum_url__,
+                font_family=font_family,
+                font_size=font_size,
+            ))
+        self.label_legal = QLabel((
+            """
+            <div style='font-family: "{font_family}";
+                        font-size: {font_size}pt;
```

Comment:
```suggestion
                        font-size: {font_size};
```

`pt` is already included in `DialogStyle.ContentFontSize`